nixos/modules/services/network-filesystems/dave.nix

```diff
@@ -0,0 +1,143 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dave;
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  options.services.dave = {
+    enable = mkEnableOption "dave WebDAV server";
+
+    user = mkOption {
+      type = types.str;
+      default = "dave-webdav";
+      description = "User account under which dave runs.";
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "dave-webdav";
+      description = "Group under which dave runs.";
```

Comment:
```suggestion
      description = ''
        Group under which dave runs.
        
        <note><para>
          If left as the default value this user will automatically be created
          on system activation, otherwise you are responsible for
          ensuring the user exists before the dave service starts.
        </para></note>
      '';
```